### PR TITLE
MangasPROJECT presence

### PR DESCRIPTION
**Pull request by Shironep**
Presence working! (main page)
image
viewing a manga page
image
Reading
image

### CODE REVIEW

**Comment by doomlerd** [comment]
change the username to "Aya-San"

File: MangasPROJECT/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "description": {
+    "en": "MangasPROJECT - Read online and organize your reads in mangasPROJECT. Read in portuguese mangas and webtoons most reads in action, romance, fantasy, isekai and others!",
+    "pt_BR": "MangasPROJECT - Leia online e organize suas leituras na mangasPROJECT. Leia antes em português os mangás e webtoons mais lidas de ação, romance, fantasia, isekai e outros!"
+  },
+  "url": [
+    "leitor.net"
+  ],
+  "version": "1.1.1",
+  "logo": "https://leitor.net/assets/images/favicon.png",
+  "thumbnail": "https://i.imgur.com/PGvPuOd.png",
```
Comment (Bas950) [inline MangasPROJECT/dist/metadata.json]: Version 1.0 (since its first release)
Upload logo to imgur.

File: MangasPROJECT/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "id": "256422547556401152"
+  },
+  "service": "MangasPROJECT",
+  "description": {
+    "en": "MangasPROJECT - Read online and organize your reads in mangasPROJECT. Read in portuguese mangas and webtoons most reads in action, romance, fantasy, isekai and others!",
+    "pt_BR": "MangasPROJECT - Leia online e organize suas leituras na mangasPROJECT. Leia antes em português os mangás e webtoons mais lidas de ação, romance, fantasia, isekai e outros!"
+  },
+  "url": [
+    "leitor.net"
+  ],
```
Comment (Bas950) [inline MangasPROJECT/dist/metadata.json]: url shouldnt be in an array if its only 1

**Comment by Shironep** [comment]
done, i think

File: MangasPROJECT/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "name": "Aya-San",
+    "id": "256422547556401152"
+  },
+  "service": "MangasPROJECT",
+  "description": {
+    "en": "MangasPROJECT - Read online and organize your reads in mangasPROJECT. Read in portuguese mangas and webtoons most reads in action, romance, fantasy, isekai and others!",
+    "pt_BR": "MangasPROJECT - Leia online e organize suas leituras na mangasPROJECT. Leia antes em português os mangás e webtoons mais lidas de ação, romance, fantasia, isekai e outros!"
+  },
+  "url": "leitor.net",
+  "version": "1.1.1",
```
Comment (Bas950) [inline MangasPROJECT/dist/metadata.json]: version should be 1.0 on release

**Comment by Bas950** [comment]
Cant merge on mobile..

**Pull request merged**